KumpeHelpers/Helper Utilities/ShowAlert.swift

```diff
@@ -157,4 +157,10 @@ public class ShowAlert {
         case none
     }
 
+//MARK: AlertMessage
```

Comment:
Codacy found an issue: [MARK comment should be in valid format. e.g. '// MARK: ...' or '// MARK: - ...'](https://app.codacy.com/gh/kumpeapps/KumpeHelpers/pullRequest?prid=6395145)